pytest-selenium/pages/content.py

```diff
@@ -73,3 +77,23 @@ def click_toc_toggle_button(self):
                 return self.wait.until(
                     expected.invisibility_of_element_located(self.toc_toggle_button)
                 )
+
+    class Attribution(Region):
+        _root_locator = (By.CSS_SELECTOR, '[data-testid="attribution-details"]')
+        _attribute_status_locator = (By.CSS_SELECTOR, 'summary[aria-label="Citation/Attribution"]')
```

Comment:
it would be better to use `'[data-testid="attribution-details"] summary'` here, in case of any future copy changes.

the name of this variable is also confusing to me

Reply:
the variable could be better. 👍 

how about `_attribution_open_locator`?

Reply:
or `_attribution_toggle_locator` maybe?

Reply:
updated to _attribution_toggle_locator